lib/glimesh/api/resolvers/channel_resolver.ex

```diff
@@ -31,6 +32,23 @@ defmodule Glimesh.Api.ChannelResolver do
     end
   end
 
+  def change_title(_parent, %{channel_id: channel_id, title: title},
+        %{context: %{access: access}
+      }) do
+    with :ok <- Bodyguard.permit(Glimesh.Api.Scopes, :title, access) do
+      channel = Glimesh.ChannelLookups.get_channel(channel_id)
+
+      if String.length(title) > 250 do
+        {:error, "Title must less than 250 characters."}
+      else
```

Comment:
This validation should exist in the changeset already, I don't think we should duplicate it. You can listen for the {:error, changeset} and use a helper to spit it back out to the client.

Example: https://github.com/Glimesh/glimesh.tv/blob/dev/lib/glimesh/api/resolvers/account_resolver.ex#L38-L44